layouts/partials/contact.html

```diff
@@ -7,15 +7,15 @@
 
                 {{ .Content }}
 
-                {{ if isset .Site.Params "email" }}
+                {{ if isset .Site.Params "formspree_action" }}
 
                 <div class="heading">
                   <h3>{{ i18n "contactForm" }}</h3>
                 </div>
 
                 <div id="contact-message"></div>
 
-                <form {{ with .Site.Params.contact_form_ajax }}class="contact-form-ajax"{{ else }}{{ end }} method="post" action="https://formspree.io/{{ .Site.Params.email }}">
+                <form {{ with .Site.Params.contact_form_ajax }}class="contact-form-ajax"{{ else }}{{ end }} method="post" action="https://formspree.io/{{ .Site.Params.formspree_action }}">
```

Comment:
I prefer `formspree_action` to have all the URL:

action="{{ .Site.Params.formspree_action }}"

Reply:
Oh! Sorry, I misunderstood you comment.